.changeset/tough-crews-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/ssr': patch
+---
+
+Fix a bug where a top level 'td' tag would be removed from a server template.
```

Comment:
```suggestion
Fix a bug where server templates with attribute bindings on certain element tags like `<td>` would throw an error during server render.
```

Reply:
Should the fact that it's a top-level tag be mentioned?

```suggestion
Fix a bug where server templates with attribute bindings on certain top-level element tags like `<td>` would throw an error during server render.
```

Reply:
Oh yes! I somehow dropped the top-level part as i was rewording. The placement makes it a bit ambiguous though. How do you like
```suggestion
Fix a bug where server templates with attribute bindings on certain element tags like `<td>` used top-level would throw an error during server render.
```

Reply:
Love it!